evalml/tests/data_checks_tests/test_ts_parameters_data_check.py

```diff
@@ -7,12 +7,33 @@
 )
 
 
-def test_time_series_param_data_check_raises_value_error():
+@pytest.mark.parametrize(
```

Comment:
You could parametrize all 4 variable individually between the values [1, None, "missing"], if you wanted complete coverage!